Tests/FluentAssertions.Specs/Collections/CollectionAssertionSpecs.BeInAscendingOrder.cs

```diff
@@ -153,6 +153,71 @@ public void When_asserting_single_element_collection_by_property_expression_orde
             act.Should().NotThrow();
         }
 
+        [Fact]
+        public void When_asserting_single_element_collection_by_converted_property_expression_ordered_in_ascending_it_should_succeed()
+        {
+            // Arrange
+            var collection = new SomeClass[]
+            {
+                new() { Text = "a", Number = 1 }
+            };
+
+            // Act
+            Action act = () => collection.Should().BeInAscendingOrder(o => (float)o.Number);
+
+            // Assert
+            act.Should().NotThrow();
```

Comment:
🔧 For scenarios that are not supposed to throw, we just call the API and that's it.